spec/request/posts_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Controllers', type: :request do
```

Comment:
- All of your request specs are passing, but with recent changes, one test is failing in spec/post_spec.rb, kindly fix that as well. This is outside of this project scope so it will be market optional but is highly recommended 
![image](https://user-images.githubusercontent.com/62705947/222761364-428cbbbc-c023-4adb-9222-ee5881f67198.png)